### PR TITLE
Blog/Entry/Picture Migration

### PR DESCRIPTION
**Pull request by DBombay**
Description
=======
- Builds backend for Blog, Entry, and Picture models
- Some testing. More to be fleshed out as I build views and controllers
- Testing for features and controllers that were auto-generated by Rails Composer are commented out until I start working on those corresponding items

Migrations
---------
**YES**

### CODE REVIEW

**Review by DRBragg — COMMENTED** [review]
neat!

**Review by DRBragg — APPROVED** [review]
Approved!

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
-
-  subject { @user }
+  it 'should create a valid user with acceptable params' do
+    expect(user).to be_valid
+    expect(user).to be_a(User)
+  end
 
-  it { should respond_to(:name) }
+  it 'should not create a user with unacceptable params' do
+    @bad_user = user
```
Comment (rjrobinson) [inline spec/models/user_spec.rb]: You should use shoulda matchers for these types of tests.

**Comment by DBombay** [comment]
dang, tried to get some small edits to the rspec file to go through before the reviews. Thanks guys!

File: spec/models/user_spec.rb (hunk 1 above)
Comment (DBombay) [reply spec/models/user_spec.rb]: added the gem. I'll re-write the tests to use shoulda-matchers

**Pull request merged**